apps/next/app/login/page.tsx

```diff
@@ -1,10 +1,36 @@
 import { redirect } from "next/navigation"
+import * as z from "zod"
```

Comment:
You can use the named export here:

```suggestion
import { z } from "zod";
```